test/WebSites/BasicWebSite/Areas/Area1/Views/RemoteAttribute_Home/Details.cshtml

```diff
@@ -64,13 +61,5 @@
 </div>
 
 <p>
-    @Html.ActionLink("Create a new one", "Create") |
-    @if (string.Equals("Aria", areaName, StringComparison.OrdinalIgnoreCase))
-    {
-        @Html.RouteLink("Go back to home", "Home")
-    }
-    else
-    {
-        @Html.RouteLink("Go to Aria home", "AriaHome")
-    }
+    @Html.ActionLink("Create a new one", "Create") | @Html.RouteLink("Go to Area1 home", "AriaHome")
```

Comment:
This change and other changes to the scenarios should be reverted. I'm :ok: with improving the folder layout and fitting into the existing areas in BasicViews.